examples/AlfaKit/Sample/Sample.ino

```diff
@@ -31,6 +31,11 @@
 void setup() {
   owl_log_set_level(L_INFO);
 
+#ifdef SAMPLE_GPS
+  // must occur before modem_setup() so GNSS serial port is associated
+  gnss_serial = new ArduinoSeeedHwOwlSerial(&SerialGNSS, SerialGNSS_Baudrate);
```

Comment:
gnss_serial can only be set at init time, previously it was created way too late and unused